common.lua

```diff
@@ -329,37 +500,10 @@ function pipeworks.create_fake_player(def, is_dynamic)
 	-- For players, move_to and get_pos do the same
 	p.move_to = p.get_pos
 
-	-- Backwards compatibilty
+	-- Backwards compatibility
 	p.getpos = p.get_pos
 	p.setpos = p.set_pos
 	p.moveto = p.move_to
-
-	-- TODO "implement" all these
-	-- set_armor_groups
-	-- get_armor_groups
-	-- get_animation
-	-- get_bone_position
-	-- get_player_velocity
-	-- set_look_pitch
-	-- set_look_yaw
-	-- set_physics_override
-	-- get_physics_override
-	-- hud_add
-	-- hud_remove
-	-- hud_get
-	-- hud_set_flags
-	-- hud_get_flags
-	-- hud_set_hotbar_itemcount
-	-- hud_get_hotbar_itemcount
-	-- hud_set_hotbar_image
-	-- hud_get_hotbar_image
-	-- hud_set_hotbar_selected_image
-	-- hud_get_hotbar_selected_image
-	-- hud_replace_builtin
-	-- set_sky
-	-- get_sky
-	-- override_day_night_ratio
-	-- get_day_night_ratio
-	-- set_local_animation
+	setmetatable(p, fake_player_metatable)
```

Comment:
Need to do this to make it work:
```suggestion
	setmetatable(p, { __index = fake_player_metatable })
```

Reply:
whoops, i know better. should be fixed in latest commit.